### ISSUE TITLE
Project name change?

### ISSUE THREAD

**Issue by leif81** [mcgredonps/open-dis#1]
To avoid conflict or confusion with the other open-dis versions for Java, python or C++, maybe this project should be called something more unique. Maybe `open-dis-js` or `open-dis.js`?

**Comment by mcgredonps** [comment]
Yeah, there's some project sprawl. The code here is actually generated over in the sourceforge release, but the node package manager wants a github presence. 

I think the ultimate solution is to just migrate the sourceforge project to github. Sourceforge has been doing some nasty stuff such as inserting their own software into prepackaged installers, and from a market share standpoint github appears to be a lot more popular these days anyway.

**Comment by leif81** [comment]
Cool. I too have grown tired of Sourceforge in more ways than one.

It might be worth considering creating an `open-dis` top level organization on github and then park all the official language repositories, etc under that.

**Comment by mcgredonps** [comment]
Exactly what I was thinking ;-)

The dependencies for each project should be handled better and more explicitly.
I’m thinking of using Ivy, but I’m not so sure that’s cross-language enough.

 I don’t think there can realistically be a unified build
system—the C++ people will want visual studio build files, the Java people maven
or ant, etc.

On Sep 18, 2015, at 7:08 PM, Leif Gruenwoldt <notifications@github.com<mailto:notifications@github.com>> wrote:

Cool. I too have grown tired of Sourceforge in more ways than one.

It might be worth considering creating an open-dis top level organization on github and then park all the official language repositories, etc under that.

**Comment by leif81** [comment]
> The dependencies for each project should be handled better and more explicitly. I’m thinking of using Ivy, but I’m not so sure that’s cross-language enough.

It's probably worth using whatever the preferred dependency manager for each language is (e.g. pip for Python, Maven for Java, npm or bower for Javascript).

> I don’t think there can realistically be a unified build system—the C++ people will want visual studio build files, the Java people maven or ant, etc. 

Ya I agree. Making it unified across languages may be more trouble than it's worth.

**Comment by leif81** [comment]
To help close the loop on this one, the new repo for this is over https://github.com/open-dis/open-dis-javascript

@mcgredonps If the other repo in the open-dis group has all the changes from this one, maybe we close this one.
